stroeer/page/article/v1/article_page.proto

```diff
@@ -28,4 +29,7 @@ message ArticlePage {
   //     - submenu
   //     - external references
   stroeer.core.v1.Reference navigation_menu = 4;
+
+  // meta information to render an article page such as commercial settings.
+  stroeer.page.section.v1.SectionPage.Section section = 5;
```

Comment:
This looks wrong to me. The equivalent of `SectionPage.Section.fields` is `ArticlePage.Article.fields`

Reply:
that is true, and I also thought about just adding the fields, but since there are efforts to make ArticlePage and SectionPage models more similar, I preferred to use the whole Section field.
@bobaaaaa

Reply:
To quote Imogen Heap: "Where do we go from here":
So just the fields than or is something wrong with that too? Since we want to keep sections and articles more alike.